### PR TITLE
Kubeone all in one Node guide

### LINKED ISSUE #707: KubeOne: Single Master/Worker
In some use-case you only have one server/VM and want to manage the workload with K8s.

KubeOne can already setup this kind of machine. We only need to overwrite the taint `NoSchedule` in the config.

Create a guide how to setup a single node as master & worker node.

### PR DESCRIPTION
**Pull request by kron4eg**
Fixes #707

### CODE REVIEW

File: content/kubeone/master/guides/all_in_one_node/_index.en.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+enableToc = true
++++
+
+Sometimes it's needed to create a one-node-cluster, which will host control
+plane components and as well the usual workloads. It's possible to do so with
+small tweaks.
+
+The setup of "one node fits all" or all-in-one in general is a usual KubeOne
+cluster, the only difference would be: removed [taints][taints-glossary] from the Node.
+
```
Comment (xmudrii) [inline content/kubeone/master/guides/all_in_one_node/_index.en.md:13]: We should put a link to the main tutorial for reference.

File: content/kubeone/master/guides/all_in_one_node/_index.en.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+## What
```
Comment (xmudrii) [inline content/kubeone/master/guides/all_in_one_node/_index.en.md:14]: Nit: let's use some more descriptive headers. It makes it easier for readers but also helps with SEO.

Example: Introduction, default setup...

File: content/kubeone/master/guides/all_in_one_node/_index.en.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+Nodes.
+
+```yaml
+- effect: NoSchedule
+  key: node-role.kubernetes.io/master
+```
+
+So we need to get rid of them.
+
+## How
```
Comment (xmudrii) [inline content/kubeone/master/guides/all_in_one_node/_index.en.md:27]: Nit: same as the other comment.

Example: How to remove the default control plane taint?

File: content/kubeone/master/guides/all_in_one_node/_index.en.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+In order to remove the default tains from control plane Node we need to ether
+use terraform output (in case when it's in use) or to specify empty tains array
+in YAML KubeOne config.
```
Comment (xmudrii) [inline content/kubeone/master/guides/all_in_one_node/_index.en.md:31]: `YAML KubeOne config` sounds a bit weird.

File: content/kubeone/master/guides/all_in_one_node/_index.en.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+### Drop control plane taints using terraform output
+
```
Comment (xmudrii) [inline content/kubeone/master/guides/all_in_one_node/_index.en.md:34]: Let's explicitly say that you need to modify the `output.tf` file (for completeness).

**Review by xmudrii — CHANGES_REQUESTED** [review]

**Comment by kron4eg** [comment]
@xmudrii PTAL again.

**Review by xmudrii — APPROVED** [review]
/lgtm
/approve

**Pull request merged**